mailer.rst

```diff
@@ -93,6 +93,11 @@ native        ``native://default``                      Mailer uses the sendmail
                                                         in the ``sendmail_path`` setting of ``php.ini``. On Windows
                                                         hosts, Mailer fallbacks to ``smtp`` and ``smtp_port``
                                                         ``php.ini`` settings when ``sendmail_path`` is not configured.
+                                                        Be warned that if ``php.ini`` uses the ``sendmail -t`` command,
```

Comment:
Maybe it's better to move this paragraph under a

.. warning:: 

Directive?

Reply:
Yes ... I've moved this into an admonition while merging. Thanks.